classes/wp-background-process.php

```diff
@@ -606,7 +651,7 @@ protected function handle() {
 		 * @param int $seconds
 		 */
 		$throttle_seconds = max(
-			0,
+			$this->seconds_between_batches,
```

Comment:
I think you probably meant for this change to apply to the value passed into the inner `apply_filters`?